generic/tests/netperf.py

```diff
@@ -216,6 +233,21 @@ def _pin_vm_threads(vm, node):
     prepare_list = set([server_ctl, client, host])
     tag_dict = {server_ctl: "server", client: "client", host: "host"}
     ip_dict = {server_ctl: server_ctl_ip, client: client_ip, host: host_ip}
+
+    mtu = params.get("mtu", "1500")
```

Comment:
Why do we need to get mtu again here as it has been already got at line128?

Reply:
Thanks, updated.